pages/chaoss/visualizations/contrib_importance_over_time.py

```diff
@@ -0,0 +1,443 @@
+from dash import html, dcc, callback
+import dash
+from dash import dcc
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.contributors_query import contributors_query as ctq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import datetime as dt
+from scipy import stats
+
+PAGE = "chaoss"  
+VIZ_ID = "contrib-importance-over-time"  
+
+gc_contrib_importance_over_time = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    id=f"graph-title-{PAGE}-{VIZ_ID}",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody("""
+                                        Visualizes how contributor importance evolves over a specified time period,\n
+                                        window width, and step size according to a specific action type. Contributor\n
+                                        importance quantifies the amount of contributors that contribute at least a\n
+                                        threshold amount of the total number of contributons. By default, the time\n
+                                        period is set to the entire lifetime of the project with a window width of 6\n
+                                        months and step size of 4 months. The interpretation of each data point is 
+                                        the contributor importance during the specified window width, starting from\n
+                                        the date indicated on the x-axis. The preceeding and proceeding data point is\n
+                                        determined by the step size.
+                                        """),
+                    ],
+                    id=f"popover-{PAGE}-{VIZ_ID}",
+                    target=f"popover-target-{PAGE}-{VIZ_ID}",  # needs to be the same as dbc.Button id
+                    placement="top",
+                    is_open=False,
+                ),
+                dcc.Loading(
+                    dcc.Graph(id=f"{PAGE}-{VIZ_ID}"),
+                ),
+                dbc.Form(
+                    [   dbc.Row(
+                            [ 
+                                dbc.Label(
+                                    "Action Type:",
```

Comment:
Change organization on inputs: 
row 1: Action type and "Contributor substring filter" (shorter name= more space) (potentially remove action type all together, see other comments)
row 2: threshold 
row 3: Window width and step size (look at staling graph for how to line them up)
row 4: Date range and about graph button